packages/epo-widget-lib/src/charts/Viewport/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { FunctionComponent, PropsWithChildren } from "react";
+
+export interface ViewportProps {
+  innerWidth: number;
+  innerHeight: number;
+  outerWidth: number;
+  outerHeight: number;
+  x: number;
+  y: number;
+}
+
+const Viewport: FunctionComponent<PropsWithChildren<ViewportProps>> = ({
```

Comment:
handy utility to make new viewports inside of SVG's for children of charts that don't need to know about their parent's margins